parcel/Example_Frostenberg_Immersion_Freezing.jl

```diff
@@ -0,0 +1,175 @@
+qᵢ = FT(0)
+x_sulph = FT(0.01)
+
+# Moisture dependent initial conditions
+q = TD.PhasePartition.(qᵥ + qₗ + qᵢ, qₗ, qᵢ)
+R_v = TD.Parameters.R_v(tps)
+Rₐ = TD.gas_constant_air(tps, q)
+eₛ = TD.saturation_vapor_pressure(tps, T₀, TD.Liquid())
+e = eᵥ(qᵥ, p₀, Rₐ, R_v)
+Sₗ = FT(e / eₛ)
+IC = [Sₗ, p₀, T₀, qᵥ, qₗ, qᵢ, Nₐ, Nₗ, Nᵢ, x_sulph]
+
+# Simulation parameters passed into ODE solver
+w = FT(0.7)                                # updraft speed
+const_dt = FT(1)                           # model timestep
+t_max = FT(1200)
+aerosol = CMP.Illite(FT)
+condensation_growth = "Condensation"
+deposition_growth = "Deposition"
+DSD = "Monodisperse"
+
+# Plotting
+fig = MK.Figure(resolution = (900, 700))
+ax1 = MK.Axis(fig[1, 1], ylabel = "Ice Supersaturation [-]")
+ax2 = MK.Axis(fig[1, 2], ylabel = "Temperature [K]")
+ax3 = MK.Axis(fig[2, 1], ylabel = "q_ice [g/kg]")
+ax4 = MK.Axis(fig[2, 2], ylabel = "q_liq [g/kg]")
+ax5 = MK.Axis(fig[3, 1], xlabel = "Time [min]", ylabel = "N_liq")
+ax6 = MK.Axis(fig[3, 2], xlabel = "Time [min]", ylabel = "N_ice")
+
+colors = [:blue, :green, :darkorange]
+
+function plot_results(
+    sol,
+    variable,
+    label = "",
+    unit = "",
+    linestyle = :solid,
+    color = :black,
+)
+
+    MK.lines!(
+        ax1,
+        sol.t / 60,
+        S_i.(tps, sol[3, :], sol[1, :]) .- 1,
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax2,
+        sol.t / 60,
+        sol[3, :],
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax3,
+        sol.t / 60,
+        sol[6, :] * 1e3,
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax4,
+        sol.t / 60,
+        sol[5, :] * 1e3,
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax5,
+        sol.t / 60,
+        sol[8, :],
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax6,
+        sol.t / 60,
+        sol[9, :],
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+
+    MK.axislegend(ax2, position = :lt)
+end
+
+#Frostenberg_mean
+params = parcel_params{FT}(
+    const_dt = const_dt,
+    w = w,
+    aerosol = aerosol,
+    heterogeneous = "Frostenberg_mean",
+    condensation_growth = condensation_growth,
+    deposition_growth = deposition_growth,
+    size_distribution = DSD,
+)
+# solve ODE
+sol = run_parcel(IC, FT(0), t_max, params)
+plot_results(sol, "mean")
+
+
+# Frostenberg_random with different drawing frequencies
+drawing_interval_range = range(FT(1), stop = FT(5), length = 3)
+
+for (drawing_interval, color) in zip(drawing_interval_range, colors)
+
```

Comment:
Once we have random seed as one of the parameters, could you loop over a couple of different random seed realizations here and plot the avarage of them. I imagine there is some (maybe a lot) of variability here and plotting an ensemble average should clarify what is happening